rspec-retry.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |gem|
   gem.name          = "rspec-retry"
   gem.require_paths = ["lib"]
   gem.version       = RSpec::Retry::VERSION
-  gem.add_runtime_dependency(%{rspec-core}, '>3.3', '<=3.5')
+  gem.add_runtime_dependency(%{rspec-core}, '>3.3', '<3.6')
```

Comment:
It might be more future-proof to go back to '~> 3.3'. Is there a reason to constraint the maximum rspec version?